panels/UserPanel.php

```diff
@@ -116,4 +124,19 @@ protected function dataToString($data)
 
         return VarDumper::export($data);
     }
+
+    /**
+     * Returns the array/object that should be set on [[\yii\widgets\DetailView::model]]
+     *
+     * @param IdentityInterface $identity
+     * @return array|object
+     */
```

Comment:
The current behavior is to only convert ActiveRecord objects to arrays, but leave other objects in-tact. There's no technical reason that a subclass couldn’t decide to override and still return an object, so I made room for that in the `@return` doc.

That said, if you think we should be encouraging subclasses to always have the method return an array, I can change the type to just `array`.